xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-ui/src/main/resources/XWiki/Notifications/Code/Translations.xml

```diff
@@ -240,6 +240,55 @@ notifications.switches.tooltip.space=Watch the parent page and its children
 notifications.switches.tooltip.wiki=Watch this wiki
 notifications.switches.changeStatusError=Failed to change the status of the notifications.
 
+## Watch button and modal
+notifications.watch.button.status.notset=Not set
+notifications.watch.button.status.followed=Followed
+notifications.watch.button.status.blocked=Blocked
+notifications.watch.button.status.custom=Custom
+
+notifications.watch.modal.title=Watch settings
+notifications.watch.modal.description.WATCHED_FOR_ALL_EVENTS_AND_FORMATS=This page is followed: you will receive notifications for all event types.
```

Comment:
Is the use of the passive voice intentional? Why not something like "You're following this page"?